libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -47,7 +47,6 @@ using namespace solidity;
 template <typename T, typename... Args>
 ASTPointer<T> ASTJsonImporter::createASTNode(Json::Value const& _node, Args&&... _args)
 {
-	astAssert(_node.isMember("id") && _node.isMember("src"), "'id' or 'src' field missing");
```

Comment:
I figured it makes more sense to check for them closer to where they are actually used. 
"src" gets checked in createSourceLocation, which is called right below.

I decided to use the "id" field in the error-message that is thrown when a json-AST-Node-object does not have a field that is required (accessed via member()). That way one can directly see why the AST can not be imported. But as the member-function is called before createASTNode(), I needed to check for it earlier, so now that happens in convertJsonToASTNode(),...........and I just realized that this is not the only route create can be called so I do need to check for it there.